src/__tests__/sheet-test.tsx

```diff
@@ -43,9 +43,10 @@ test('Sheet', async () => {
 
```

Comment:
I don't know why this test suite is so slow and requires extra timeouts. 
Ticket to investigate: https://jira.tid.es/browse/WEB-1911

Expect some instability in this suite, I've removed/reduced the timeouts. "It works on my machine"

Reply:
I think it's because: big dom tree + slow findByRole